qdrant-landing/content/documentation/private-cloud/qdrant-cluster-management.md

```diff
@@ -195,3 +195,92 @@ spec:
           name: qdrant-tls
           key: tls.key
 ```
+
+### Configuring TLS for Inter-cluster Communication
+
+*Available as of Operator v2.2.0*
```

Comment:
This needs a warning level note, that right now p2p tls can only be set up, if you create a cluster. But if you update existing clusters, it will break communication between cluster nodes.

Reply:
Updating the existing cluster is not possible and users get `phase: FailedUpdate` and a proper reason. We can though mention that updating existing cluster is not possible.